### DISCUSSION TITLE
Seeking participants for api.weather.gov improvements research

### DISCUSSION THREAD

**Discussion by greg-does-weather** [weather-gov/api#784] [General]
Hello! My name is Greg and I work with [18F](https://18f.gsa.gov), a federal agency that NWS has partnered with to help spruce up api.weather.gov. The NWS team has done fantastic work over the years and they have cool ideas for the future, and we're very excited and honored to get to work with them.

The first part of our work is to talk directly to users to understand their needs. To that end, we're seeking volunteers to tell us about their uses and any challenges they've faced when using the API. These interviews are conducted over video and last around an hour or less, and you will get to schedule it yourself to best fit your availability. If video doesn't work for you, there will also be a provided phone number so you can simply call in.

We've analyzed this discussion board and identified a few users we think may have a lot to say, and I'm going to @-mention them here. However, anyone should feel free to respond to this post. Unfortunately we can't interview everyone, so we'll pick a small number of people based on those who show interest. We'll add a comment on this thread once we've picked our participants along with a private link that will let you schedule your interview.

Thanks in advance to this community for any and all help you're able to provide us on this effort!

---

cc:
- @jjkraw
- @dwhitemv25
- @ktrue
- @netbymatt
- @akrherz

**Comment by ktrue** [comment]
I'd be glad to participate.  
Ken

**Comment by dwhitemv25** [comment]
I'm in to help.

**Comment by joepperkins** [comment]
You can count me in.

**Comment by netbymatt** [comment]
I'd be glad to help out.

**Comment by paulnadler** [comment]
I've been using this service to provide data for a large number of users since early 2017, during the initial migration to forecast.weather.gov (which was rolled back). Happy to provide feedback.

**Comment by greg-does-weather** [comment]
Thanks everyone for raising your hands! You should all have received an invitation for another GitHub repo. From there, you'll find instructions for how you can sign up for an interview slot. We really appreciate your help and look forward to chatting with each of you.

---

Because of the fast-approaching holiday season, we're not going to send out any additional invitations below this line for now, but we'll keep the discussion board open. If you're interested in participating and haven't already commented, feel free to comment below and we'll reach out to you if there's a second round of interviews. There will also likely be additional interviews in the spring as work gets underway on implementing changes, and we'll reach out then as well.

Thanks again!
